add utila story (#163)
metalbear-co/mirrord.dev: content/stories/utila/index.md

---
title: "Utila User Story | Roman Yarinovsky"
card_title: "Roman Yarinovsky"

> description: "mirrord seamlessly integrating into our current environment and allow us to debug remotely is a game changer for us"

Reviewer: ```suggestion
description: "mirrord seamlessly integrating into our current environment and allowing us to debug remotely is a game changer for us"

```

date: 2023-09-19T06:00:00+00:00
lastmod: 2023-09-19T06:00:00+00:00

> position: "Senir Software Engineer | Utila"

Reviewer: ```suggestion
position: "Senior Software Engineer | Utila"

```

avatar: "roman_yarinovsky.png"
logo: "utila.png"
featured: true
draft: false
weight: 61
---

## About yourself

> Roman Yarinovsky, Senior Software engineer

Reviewer: ```suggestion
Roman Yarinovsky, Senior Software Engineer

```

## About your company

Utila Wallet is an out-of-the-box platform that lets you securely store, transfer and take complete control of your digital assets.

## What did you use before?

Nothing.

## Why did you choose mirrord instead?

> Up until this point we did not have remote debugging, mostly because we didn't have the time to invest in implementing a solution, and having mirrord seamlessly integrating into our current environment and allow us to debug remotely is a game changer for us.

Reviewer: ```suggestion
Up until this point we did not have remote debugging, mostly because we didn't have the time to invest in implementing a solution, and having mirrord seamlessly integrating into our current environment and allowing us to debug remotely is a game changer for us.

```

The fact that the setup is so minimalistic, and that it supports a polyglot project, again, seamlessly.

> Also important to note it was very simple to set up the vscode extension and have it running.

Reviewer: ```suggestion
Also important to note it was very simple to set up the VS Code extension and have it running.

```
